release_notes/v2_vs_v3.adoc

```diff
@@ -16,6 +16,10 @@ Many of the same terms are used, and the same functions are performed, but the t
 
 This topic discusses these differences in detail, in an effort to help OpenShift users in the transition from OpenShift v2 to OpenShift v3.
 
+ifdef::openshift-dedicated[]
+OpenShift Dedicated uses the same code base as OpenShift Enterprise 3. OpenShift Dedicated is hosted on the public cloud and managed by Red Hat. An OpenShift Enterprise 3 subscription entitles you to host and manage the software on your infrastructure of choice.
+endif::[]
```

Comment:
There's an earlier use of v2/v3 in the first para/sentence above this, if you were wanting to spell it out in their first appearance.